### ISSUE TITLE
Support dynamic markdown

### ISSUE THREAD

**Issue by Steve-Mcl** [flowforge/flowforge-nr-dashboard#91]
### Description

Static markdown if great but permitting runtime updates would make the markdown node great for building live reports and rich, up-to-date/dynamic content.

<br>

As I see it, there are 2 options available to us:

1. permit the `msg` to provide the markdown - meaning we would use the built-in Node-RED `template` node to generate markdown with mustache/handlebars replacements.

     OR

2. Support handlebars replacements in the markdown input e.g

Option 1 means less work and maintenance in the code base - but not intuitive for the user
Option 2 means one less template node and no separation of markdown and dynamic element (more intuitive)

<br>

### Entering this markdown:
```
#### {{name}} Status

##### OEE

Performance: {{performance}}
Availability: {{availability}}
Quality: {{quality}}

OEE = {{performance}} ✕ {{availability}} ✕ {{quality}} 
OEE = {{OEE}}

##### Build

Part count: {{partCount}}
Part target: {{targetCount}}
Scrap Count: {{scrap}}

```
<br>

### And given this payload:
```
{
  name: "Cooling Stage 1"
  time: '16:00',
  cycleCount: 800,
  availability: 0.938,
  performance: 0.933,
  quality: 0.983
  OEE, "86%"
  productionHours: 8,
  partCount: 700,
  targetCount: 800,
  scrap: 12
}
```

<br>

##### OEE

Performance: 0.933
Availability: 0.938
Quality: 0.983

OEE = 0.933 ✕ 0.938 ✕ 0.983
OEE = 86%

##### Build

Part count: 700
Part target: 800
Scrap Count: 12

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by Steve-Mcl** [comment]
NOTE: Where I state hanlebars/mustach, we could just as well support any of the popular templating formats

e.g.
* [MARKO ](https://markojs.com/)
* [JADE (PUG)](https://pugjs.org/api/getting-started.html)
* [EJS](https://github.com/mde/ejs#example)
* [NUNJUCKS](https://mozilla.github.io/nunjucks/)


It may even be possible / desirable to use VUE/VUEX data reactively - now that would be cool!

**Comment by joepavitt** [comment]
Challenge here is where does the reactivity occur? We'd need to parse the mustache server-side, as I don't believe the client-side would be able to parse this _within_ the markdown.

Doing it server-side also gives us access to evaluated variables 9e.g. `flow.` or `context.`, but it doesn't natively support two-way binding/updates.

**Comment by Steve-Mcl** [comment]
I think sever side rendering will be ok so long as a msg pushed into to the node re-evaluates / rerenders and updates the front-end without a refresh. 

One thing to consi

**Comment by joepavitt** [comment]
Closed by #126
